docker/testcafe-docker.sh

```diff
@@ -6,4 +6,5 @@ dbus-daemon --session --fork
 Xvfb :1 -screen 0 "${XVFB_SCREEN_WIDTH}x${XVFB_SCREEN_HEIGHT}x24" >/dev/null 2>&1 &
 export DISPLAY=:1.0
 fluxbox >/dev/null 2>&1 &
-node /usr/lib/node_modules/testcafe/bin/testcafe-with-v8-flag-filter.js --ports 1337,1338 "$@"
+cd /usr/lib/node_modules/testcafe
+node bin/testcafe-with-v8-flag-filter.js --ports 1337,1338 "$@"
```

Comment:
We cannot change the current working directory here. CI subsystems set the current working directory to the project root before starting TestCafe, and users may expect that the project root is the current directory when using relative file names in test files.